src/m365/planner/commands/task/task-get.ts

```diff
@@ -0,0 +1,56 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Task } from '../../Task';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+}
+
+class PlannerTaskGetCommand extends GraphItemsListCommand<Task> {
```

Comment:
Since we're retrieving just one item, we shouldn't use the `GraphItemsListCommand` base class, but rather `GraphCommand`